nautilus_trader/test_kit/rust/identifiers_pyo3.py

```diff
@@ -100,7 +100,7 @@ def order_list_id(counter: int = 1) -> OrderListId:
 
     @staticmethod
     def venue_order_id() -> VenueOrderId:
-        return VenueOrderId("1")
+        return VenueOrderId("123456")
```

Comment:
Small thing, but I think we're using low numbers for `VenueOrderId` stubs in the Python codebase, e.g. `1` as they often match example API docs, and also makes debugging logs _slightly_ more readable because we often use `O-123456` for client order IDs.